gui/wxpython/lmgr/frame.py

```diff
@@ -337,13 +354,9 @@ def _createNoteBook(self):
             name='catalog')
 
         # create displays notebook widget
-        self.notebookLayers = GNotebook(parent=self.notebook,
-                                        style=globalvar.FNPageStyle)
-        self.notebookLayers.SetTabAreaColour(globalvar.FNPageColor)
-        menu = self._createTabMenu()
-        self.notebookLayers.SetRightClickMenu(menu)
+        self.display_panel, self.notebookLayers = self._createDisplayPanel()
         self.notebook.AddPage(
-            page=self.notebookLayers,
+            page=self.display_panel,
```

Comment:
Please just double check that this is the only place where this replacement is needed.

Reply:
I think it is all right. display_panel is just a widget "wrapper".